operations/validate.go

```diff
@@ -14,6 +15,10 @@ import (
 	yaml "gopkg.in/yaml.v2"
 )
 
+const (
+	localModuleFlagName = "module"
```

Comment:
Can you define this in the same place as longFlagName?

Reply:
done